docs/user-guide/grpc.md

```diff
@@ -8,102 +8,97 @@ Ambassador makes it easy to access your services from outside your application.
 
 This tutorial assumes you have already followed the [Ambassador Getting Started](/user-guide/getting-started.html) guide. If you haven't done that already, you should do that now.
 
-After completing [Getting Started](/user-guide/getting-started.html), you'll have a Kubernetes cluster running Ambassador and the Quote of the Moment service. Let's walk through adding the [Hello World gRPC service](https://github.com/grpc/grpc-go/tree/master/examples/helloworld) for this tutorial. 
+After completing [Getting Started](/user-guide/getting-started.html), you'll have a Kubernetes cluster running Ambassador and the Quote of the Moment service. Let's walk through adding an example [echo gRPC service](https://github.com/datawire/grpc-example) for this tutorial. 
 
 ## Mapping gRPC Services
 
 Ambassador `Mapping`s are based on URL prefixes; for gRPC, the URL prefix is the full service name, including the package path. 
 
-For `Hello World`, in its [proto definition file](https://github.com/grpc/grpc-go/blob/master/examples/helloworld/helloworld/helloworld.proto), we see
+For `Hello World`, in its [proto definition file](https://github.com/datawire/grpc-example/blob/master/helloworld/helloworld.proto), we see
```

Comment:
I found how you mixed Hello World and "echo gRPC service" a bit confusing. Picking a standard nomenclature would help.